src/components/popover-container/popover-container-test.stories.tsx

```diff
@@ -97,3 +104,131 @@ export const PopoverContainerWithSelect = () => {
     </div>
   );
 };
+
+export const InAScrollableBlock = () => {
```

Comment:
**question** (non-blocking): I'm not really sure what value this test story is adding, or what its relevance is to the PR since it doesn't use the new prop you've added. (But I'm fine with keeping it if we do want to manually test how PopoverContainer behaves in a scrollable container - I understand the original issue arise from a similar scenario.)

Reply:
Yeah, my thinking was we can have an example of it in a scrollable block so we can manually test for it in future. If you think I can improve the example, feel free to add suggestions and I'll get the changes made 😄